### PR TITLE
Add English Generator (AWS)

### PR DESCRIPTION
**Pull request by Vboivin**
I've added:

- [x] USAGE.md docs if applicable
- [x] CHANGELOG.md

This PR is related to #336

It slightly modifies #336 by providing the full name for each region and by adding AWS services to Faker.

### CODE REVIEW

File: lib/faker/aws/en.ex [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    "Canada (Central)",
+    "Europe (Frankfurt)",
+    "Europe (Ireland)",
+    "Europe (London)",
+    "Europe (Milan)",
+    "Europe (Paris)",
+    "Europe (Stockholm)",
+    "Middle East (Bahrain)",
+    "South America (São Paulo)"
+  ])
```
Comment (anthonator) [inline lib/faker/aws/en.ex:45]: I'm curious why you decided to use the long region names instead of region codes here.

Comment (anthonator) [reply lib/faker/aws/en.ex:45]: Why I ask is because I typically default to region codes when thinking of AWS regions rather than the name.

Comment (Vboivin) [reply lib/faker/aws/en.ex:45]: It is biased toward my own use, I use faker to generate random data that is human-readable. I could introduce a function in another PR to provide the user with the choice between calling Faker.Aws.En.region/0 and Faker.Aws.En.region_code/0. The second function would generate random data using the region codes. It is an implementation detail but I could also transform Faker.Aws.En.region/0 to Faker.Aws.En.Region/1 and if the user wants the region code they could ask it by passing an argument.

Comment (anthonator) [reply lib/faker/aws/en.ex:45]: Thanks for the context. Would you be willing to rename this function to `region_name` and introduce a `region_code` function in either this or a subsequent pull request? I know I'd get tripped up on this.

Comment (Vboivin) [reply lib/faker/aws/en.ex:45]: 100%, I will PR something later today after 5 PM (GMT-4)

Comment (anthonator) [reply lib/faker/aws/en.ex:45]: Awesome! Thank you!

**Comment by Vboivin** [comment]
@anthonator Everything has been updated according to what we discussed :heavy_check_mark:

**Comment by anthonator** [comment]
Awesome! Thank you!

**Pull request merged**